Looped/Interface/Create/CreateLibrary/CreateLibraryViewController.swift

```diff
@@ -0,0 +1,12 @@
+import UIKit
+
+final class CreateLibraryViewController: ViewController {
+    
+    // MARK: - Properties
+    let createLibraryView = CreateLibraryView()
```

Comment:
to powinno być onaczone jako prywatne

Reply:
ahhh... bo public jest `view` tej klasy..?

Reply:
`guard let createOnlineView = createOnlineViewController.view else { return }`

Reply:
nie no, `view` jako property view controllers bedzie widocznie, nie bedzie natomiast `createOnlineView` bo de facto nie jest to potrzebne

Reply:
Yep